bouncer/util.py

```diff
@@ -29,27 +29,42 @@ def __str__(self):
 
 def parse_document(document):
     """
-    Return the ID and URI from the given Elasticsearch annotation document.
+    Return the ID, URI, quote, and text from the given Elasticsearch
+    annotation document.
 
     Return the annotation ID and the annotated document's URI from the given
     Elasticsearch annotation document.
 
+    Also return quote (if available) and text to enhance the share card.
+
     :param document: the Elasticsearch annotation document to parse
     :type document: dict
 
     :rtype: 2-tuple of annotation ID (string) and document URI (string)
 
     """
     # We assume that Elasticsearch documents always have "_id" and "_source".
+
     annotation_id = document["_id"]
     annotation = document["_source"]
+    quote = ""
+    text = ""
 
     document_uri = None
 
+    if "text" not in annotation:
+        text = "Follow this link to see the annotation on the original page."
+    else:
+        text = annotation["text"]
```

Comment:
Personal opinion, no need to change if you disagree: You can make setting the `text` variable a bit more straight forward doing:
```
text = annotation.get('text', 'Follow this link to see the annotation on the original page.')
```
And at the same time you can remove the empty assignment on line 51 (`text = ""`).